packages/webpack/test/e2e-scuttle.spec.js

```diff
@@ -0,0 +1,87 @@
+const test = require('ava')
+const path = require('node:path')
+// eslint-disable-next-line ava/no-import-test-files
+const { scaffold, runScriptWithSES } = require('./scaffold.js')
+const {makeConfig} = require('./fixtures/main/webpack.config.js')
+
+const err = (intrinsic) =>
+  'LavaMoat - property "' +
+  intrinsic +
+  '" of globalThis is inaccessible under ' +
+  'scuttling mode. To learn more visit https://github.com/LavaMoat/LavaMoat/pull/360.'
+
+async function scuttle(t, scuttleGlobalThis, globals) {
+  const webpackConfigDefault = makeConfig({
+    scuttleGlobalThis,
+    generatePolicy: true,
+    diagnosticsVerbosity: 1,
+    policyLocation: path.resolve(__dirname, 'fixtures/main/policy-scuttling'),
```

Comment:
(suggestion) `__dirname` is absolute so `path.resolve` can be `path.join`.

Reply:
this is a copy-paste from the rest of the code, should we maybe address all occurrences in a separate or?

Reply:
it's very minor